digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/MunicipalityDao.scala

```diff
@@ -82,4 +82,10 @@ class MunicipalityDao {
       sql"""select ELY_NRO from municipality  where id in (#${municipalities.mkString(",")} ) group by ELY_NRO""".as[Int].list
     }
   }
+
+  def getElysIdAndNamesByCode(elys: Set[Int]): Seq[(Int, String)] ={
```

Comment:
delete this method. It is not needed

Reply:
After talking with the team the development is correct